### PR TITLE
metadata-query code optimization

### PR DESCRIPTION
**Pull request by ChengJie1053**
metadata-query code optimization

### CODE REVIEW

File: linkis-public-enhancements/linkis-datasource/linkis-metadata-query/common/src/main/java/org/apache/linkis/metadata/query/common/cache/CacheConfiguration.java [hunk 1]
```diff
@@ -35,4 +35,9 @@ public class CacheConfiguration {
       CommonVars.apply(
           "wds.linkis.server.mdq.mysql.relationship",
           "mysql,oracle,kingbase,postgresql,sqlserver,db2,greenplum,dm,doris,clickhouse,tidb,starrocks,gaussdb");
+
+  public static final CommonVars<String> QUERY_DATABASE_RELATIONSHIP =
+      CommonVars.apply(
+          "wds.linkis.server.mdq.query.database.relationship",
```
Comment (peacewong) [inline linkis-public-enhancements/linkis-datasource/linkis-metadata-query/common/src/main/java/org/apache/linkis/metadata/query/common/cache/CacheConfiguration.java:41]: new conf should to remove wds

File: linkis-public-enhancements/linkis-datasource/linkis-metadata-query/server/src/main/java/org/apache/linkis/metadata/query/server/loader/MetaClassLoaderManager.java [hunk 2]
```diff
@@ -128,7 +130,26 @@ public BiFunction<String, Object[], Object> getInvoker(String dsType) throws Err
                 }
                 String expectClassName = null;
                 if (dsType.length() > 0) {
-                  String prefix = dsType.substring(0, 1).toUpperCase() + dsType.substring(1);
+                  String converDsType = dsType;
+                  try {
+                    HashMap map =
```
Comment (peacewong) [inline linkis-public-enhancements/linkis-datasource/linkis-metadata-query/server/src/main/java/org/apache/linkis/metadata/query/server/loader/MetaClassLoaderManager.java:135]: Can be modified as a static variable?

File: linkis-public-enhancements/linkis-datasource/linkis-metadata-query/common/src/main/java/org/apache/linkis/metadata/query/common/cache/CacheConfiguration.java (hunk 1 above)
Comment (casionone) [reply linkis-public-enhancements/linkis-datasource/linkis-metadata-query/common/src/main/java/org/apache/linkis/metadata/query/common/cache/CacheConfiguration.java:41]: update docs/configuration

Comment (ChengJie1053) [reply linkis-public-enhancements/linkis-datasource/linkis-metadata-query/common/src/main/java/org/apache/linkis/metadata/query/common/cache/CacheConfiguration.java:41]: > new conf should to remove wds

Ok, thank you for reviewing the code

Comment (ChengJie1053) [reply linkis-public-enhancements/linkis-datasource/linkis-metadata-query/common/src/main/java/org/apache/linkis/metadata/query/common/cache/CacheConfiguration.java:41]: > update docs/configuration

Ok, thank you for reviewing the code

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**